README.md

```diff
@@ -62,6 +62,31 @@ Follow linked crate name for detailed status. Please note that all crates follow
 
 [semver]: https://semver.org
 
+### Pipeline Integration
+
+Some CI/CD pipelines leverage repository cloning. Below is a copy-paste-able example to build docker images for such workflows.
```

Comment:
Could you provide more context? Which pipelines are you talking about? What's the general idea on how to get from images to using them in a pipeline?

Think blank slate, and I am good test for that as I wouldn't be able to use the docker images from this description.